pkg/schedule/checker/checker_controller.go

```diff
@@ -88,7 +88,7 @@ type Controller struct {
 
 // NewController create a new Controller.
 func NewController(ctx context.Context, cluster sche.CheckerCluster, conf config.CheckerConfigProvider, ruleManager *placement.RuleManager, labeler *labeler.RegionLabeler, opController *operator.Controller) *Controller {
-	pendingProcessedRegions := cache.NewDefaultCache(DefaultPendingRegionCacheSize)
+	pendingProcessedRegions := cache.NewIDTTL(ctx, time.Minute, 3*time.Minute)
```

Comment:
sure, previous `suspectRegions` is `*cache.TTLUint64`

Reply:
Do we need to add a test for this issue?

Reply:
It's not just the problem of suspectRegions. The waiting list also has the same problem previously.

Reply:
ok, then do we need to add a test for this issue?

Reply:
why 3 mins?

Reply:
The original setting is 3 min.